app/routes/events/components/EventList.js

```diff
@@ -123,11 +123,19 @@ const EventList = (props: EventListProps) => {
     <div className={styles.root}>
       <Helmet title="Arrangementer" />
       <Toolbar actionGrant={props.actionGrant} />
-      <EventListGroup name="Denne uken" events={events.currentWeek} />
+      {events.currentWeek && (
+        <EventListGroup name="Denne uken" events={events.currentWeek} />
+      )}
 
-      <EventListGroup name="Neste uke" events={events.nextWeek} />
+      {events.nextWeek && (
+        <EventListGroup name="Neste uke" events={events.nextWeek} />
+      )}
 
-      <EventListGroup name="Senere" events={events.later} />
+      {events.later && <EventListGroup name="Senere" events={events.later} />}
+
+      {!props.events && (
+        <h2 className={styles.heading}>Ingen kommende arrangementer</h2>
```

Comment:
Here you can use the `EmptyState`  component. Example here: https://github.com/webkom/lego-webapp/blob/86f6e649ed68e8b7d15e18b7a2b2a1ca8d870f4e/app/routes/overview/components/Feed.js#L56